stonadsstatistikk/src/main/kotlin/no/nav/familie/eksterne/kontrakter/VedtakV2.kt

```diff
@@ -0,0 +1,112 @@
+package no.nav.familie.eksterne.kontrakter
+
+import java.time.LocalDate
+import java.time.YearMonth
+import java.time.ZonedDateTime
+
+data class VedtakDVHV2(val fagsakId: String,
+                       val behandlingsId: String,
+                       val tidspunktVedtak: ZonedDateTime,
+                       val personV2: PersonDVHV2,
+                       val ensligForsørger: Boolean,
+                       val kategoriV2: KategoriV2,
+                       val underkategoriV2: UnderkategoriV2,
+                       val behandlingTypeV2: BehandlingTypeV2,
+                       val utbetalingsperioderV2: List<UtbetalingsperiodeDVHV2>,
+                       val kompetanseperioder: List<Kompetanse>? = null,
+                       val funksjonellId: String? = null,
+                       val behandlingÅrsakV2: BehandlingÅrsakV2? = null,
+)
+
+data class UtbetalingsperiodeDVHV2(val hjemmel: String,
+                                   val utbetaltPerMnd: Int,
+                                   val stønadFom: LocalDate,
+                                   val stønadTom: LocalDate,
+                                   val utbetalingsDetaljer: List<UtbetalingsDetaljDVHV2>)
+
+data class UtbetalingsDetaljDVHV2(val person: PersonDVHV2,
+                                  val klassekode: String,
+                                  val utbetaltPrMnd: Int,
+                                  val delytelseId: String)
+
+
+data class PersonDVHV2(val personIdent: String,
+                       val rolle: String,
+                       val statsborgerskap: List<String>,
+                       val bostedsland: String,
+                       val delingsprosentYtelse: Int)
+
+data class Kompetanse(val barnsIdenter: List<String>,
+                      val fom: YearMonth,
+                      val tom: YearMonth?,
+                      val sokersaktivitet: SøkersAktivitet,
+                      val annenForeldersAktivitet: AnnenForeldersAktivitet? = null,
+                      val annenForeldersAktivitetsland: String? = null,
+                      val barnetsBostedsland: String? = null,
+                      val resultat: KompetanseResultat? = null
+)
+
+enum class SøkersAktivitet {
+    ARBEIDER_I_NORGE,
+    SELVSTENDIG_NÆRINGSDRIVENDE,
+    MOTTAR_UTBETALING_FRA_NAV_SOM_ERSTATTER_LØNN,
+    UTSENDT_ARBEIDSTAKER_FRA_NORGE,
+    MOTTAR_UFØRETRYGD_FRA_NORGE,
+    MOTTAR_PENSJON_FRA_NORGE,
+    INAKTIV
+}
+
+enum class AnnenForeldersAktivitet {
+    I_ARBEID,
+    MOTTAR_UTBETALING_SOM_ERSTATTER_LØNN,
+    FORSIKRET_I_BOSTEDSLAND,
+    MOTTAR_PENSJON,
+    INAKTIV,
+    IKKE_AKTUELT
+}
+
+enum class KompetanseResultat {
+    NORGE_ER_PRIMÆRLAND,
+    NORGE_ER_SEKUNDÆRLAND,
+    TO_PRIMÆRLAND
+}
+
+enum class BehandlingTypeV2(val visningsnavn: String) {
```

Comment:
Trenger vi V2 av disse. Er ingen endring fra versjon 1

Reply:
nei, strengt tatt ikke, var bare redd for at det ville bli litt vanskelig å se hvilken versjon man brukte i mappingen, når vi publiserer 2 steder en periode